client/src/store/actions/userActions.js

```diff
@@ -0,0 +1,7 @@
+import { createAction } from 'redux-actions';
+
+const editUserInfo = createAction('EDIT_USER');
```

Comment:
'EDIT_USER' can be as const in config file, so if change is required then we will know where

Reply:
createAction creates an action creator, meaning everytime we call editUserInfo(), it will create an object like that : {type:'EDIT_USER'). 
no where else do i need to write EDIT_USER, because if i need an action with type EDIT_USER i just call editUserInfo, so there is no reason to put it in a config file.
if i want to change it i can just change it where its created (much easier to find).
In general this is the way to create actions.